app/controllers/volunteer_ops_controller.rb

```diff
@@ -55,19 +55,18 @@ def volunteer_op_params
     )
   end
 
-  private
-
-  def build_map_markers(organisations)
-    ::MapMarkerJson.build(organisations) do |org, marker|
+  def build_map_markers(organisations, type = :harrow, include_extra_organisation_data = true)
+    ::MapMarkerJson.build(organisations, include_extra_organisation_data) do |org, marker|
```

Comment:
Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)